src/exceptions/OCaml-LGPL-linking-exception.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <exception licenseId="OCaml-LGPL-Linking-Exception" name="OCaml LGPL Linking Exception">
+      <crossRefs>
+         <crossRef>https://caml.inria.fr/ocaml/license.en.html</crossRef>
+      </crossRefs>
+      <notes>
+        Adopted by OCaml core in 2001 here: https://github.com/ocaml/ocaml/commit/02ef950033b81fe371759f024faa55f361ba83a6#diff-9879d6db96fd29134fc802214163b95a (git-svn-id: http://caml.inria.fr/svn/ocaml/trunk@4146 f963ae5c-01c2-4b8c-9fe0-0dff7051ff02)
```

Comment:
`<notes>` [takes `formattedFixedTextType`][1].  Do we want to add a link element (`a`) [here][2] to make this sort of thing more readable?

[1]: https://github.com/spdx/license-list-XML/blob/v3.1/schema/ListedLicense.xsd#L47
[2]: https://github.com/spdx/license-list-XML/blob/v3.1/schema/ListedLicense.xsd#L348-L355

Reply:
Maybe a good idea, but this is a bigger issue for the schema. <wontfix> as far as this PR is concerned.